internal/pkg/skuba/deployments/ssh/cri.go

```diff
@@ -52,7 +52,10 @@ func criConfigure(t *Target, data interface{}) error {
 		}
 	}
 
-	_, _, err = t.ssh("mv -f /tmp/crio.conf.d/01-caasp.conf /etc/crio/crio.conf.d/01-caasp.conf")
+	if _, _, err = t.ssh("mkdir -p /etc/crio/crio.conf.d"); err != nil {
```

Comment:
nit (optimization): This will technically upload all files, and then only copy the ones relevant. Can we not upload the README file over?